docs/react-testing-library/setup.mdx

```diff
@@ -247,44 +247,76 @@ export {
 You can then override and append the new queries via the render function by
 passing a [`queries`](api.mdx#render-options) option.
 
-If you want to add custom queries globally, you can do this by defining a custom
-render method:
+If you want to add custom queries globally, you can do this by defining your customized
+`render`, `screen` and `within` methods:
 
   <Tabs groupId="test-utils" defaultValue="jsx" values={[ {label: 'Javascript',
   value: 'jsx'}, {label: 'Typescript', value: 'tsx'}, ]}>
 
   <TabItem value="jsx">
 
 ```js title="test-utils.js"
-import {render, queries} from '@testing-library/react'
+import {render, queries, screen, within} from '@testing-library/react'
 import * as customQueries from './custom-queries'
 
+const customScreen = {
+  ...screen,
+  ...within(document.body, customQueries),
+}
```

Comment:
I think this should move below `allQueries` and then 
```suggestion
const customScreen = within(document.body, allQueries)
```

Reply:
Done